packages/renderer/src/lib/ui/TitleBar.svelte

```diff
@@ -15,11 +15,9 @@ onMount(async () => {
 
 <header
   id="navbar"
-  class="{platform === 'win32' ? 'bg-[#202020]' : 'bg-[var(--pd-titlebar-bg)]'} body-font z-[999] relative {platform ===
```

Comment:
hello, where {platform === 'win32' ? 'bg-[#202020]' } did go ?

Reply:
Hi @benoitf , I have removed it to force Windows to use (`bg-[var(--pd-titlebar-bg)]`) light/dark titlebar as the Mac/Linux

Reply:
yes but probably override the value in case of Windows to match the default colors of Windows

maybe a later improvement, we could also ask the windows registry the colors of the user to match their values

if user has title bars being like 'orange' we should use the same color ? (but it's not related to this PR)